pkg/server/cluster_server.go

```diff
@@ -129,6 +140,17 @@ func (cs *clusterServer) GetConfig(cr poolRequest) (*runtime.RawExtension, error
 		return nil, fmt.Errorf("could not get controllerconfig: %w", err)
 	}
 
+	// we cannot mock this in a test env
+	if cs.configMapLister != nil {
+		cm, err := cs.configMapLister.ConfigMaps("openshift-machine-config-operator").Get("kubeconfig-data")
```

Comment:
suggestion: Change `"openshift-machine-config-operator"` to use the `ctrlcommon.MCONamespace` constant since we already have the package imported.